SPECS/rust/rust.spec

```diff
@@ -167,6 +168,9 @@ rm %{buildroot}%{_bindir}/*.old
 %{_mandir}/man1/*
 
 %changelog
+* Tue Oct 24 2023 Rohit Rawat <rohitrawat@microsoft.com> - 1.72.0-3
```

Comment:
There seems to be a slight typo in the previous changelogs that assumes we have bumped to `1.72.2`.
Thank you for clarifying this.
/cc @dmcilvaney for visibility.